.github/workflows/openshift-nightly.yml

```diff
@@ -45,8 +45,19 @@ jobs:
           CI_MAX_WAIT_FOR_POD_TIME_SECONDS: 150
           CI_MIN_SUCCESS_THRESHOLD: 1
           OSM_HUMAN_DEBUG_LOG: true
-          PUBLISH_IMAGES: "false"
+          USE_PRIVATE_REGISTRY: "true"
```

Comment:
the openservicemesh registry is pubish, right? So this should be false, because we don't need the secret for this so we also don't need to link it for pulling images